tests/installation/bootloader_zkvm.pm

```diff
@@ -81,7 +81,11 @@ sub run {
         if (check_var("VIDEOMODE", "text")) {
             wait_serial("run 'yast.ssh'", 300) || die "linuxrc didn't finish";
             select_console("installation");
-            enter_cmd("TERM=linux yast.ssh") && record_soft_failure('bsc#1054448');
+            # If libyui REST API is used, we set it up in installation/setup_libyui
+            unless (get_var('YUI_REST_API')) {
+                enter_cmd("TERM=linux yast.ssh");
+                record_soft_failure('bsc#1054448');
+            }
```

Comment:
Seems this has introduced a soft failure for all zkvm tests where **YUI_REST_API** is not set, for example:

https://openqa.suse.de/tests/5899471 vs https://openqa.suse.de/tests/5926007

Seeing as bsc#1054448 has been closed for a while, would it be acceptable to drop this `record_soft_failure`?

Reply:
Thanks Alvaro, we were not aware, #12519.

Reply:
Thanks Joaquin.